lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -488,7 +524,8 @@ private JourneyResponse buildReuseResponse(
                 auditEventUser.getSessionId(),
                 false);
         migrateCredentialsToEVCS(userId, vcs, evcsAccessToken);
```

Comment:
I think we need to consider the interaction with migration here as well - if the user is returning from F2F then we shouldn't be running the migration process on the pending VCs (that should be covered by the new store identity step below).